web/cats/static/cats/cat_profile_style.css

```diff
@@ -1,17 +1,54 @@
+*{
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+/*Style for a header*/
```

Comment:
If it's *a* header, then you expect multiple `<header>` elements on your page (e.g. one per `<section>` or `<article>`) and they will all be pinned to the top of the page, one hidden under another, which is probably not what you want.

If it's *the* header, then perhaps it would be a good idea to ensure that the element is unique on the page. The tag does not guarantee uniqueness (AFAIK, it must be unique per parent, not per document). So it could make sense to have something like `<header id="page-header">` and select it in CSS via `#page-header` or `header#page-header` (you don't have to name it `page-header`, it's just a suggestion).

Reply:
Thank you! I chose the funny name for it, but it works for me.